src/e2e-test/features/gcs/source/GCSSourceError.feature

```diff
@@ -35,19 +35,6 @@ Feature: GCS source - Verify GCS Source plugin error scenarios
     Then Enter GCS source property path field "gcsInvalidPathField"
     Then Verify Output Path field Error Message for incorrect path field "gcsInvalidPathField"
 
-  @GCS_DELIMITED_TEST
-  Scenario: To verify Error for incorrect delimiter
```

Comment:
Is this no longer relevant?  Do you know if the behaviour has changed recently?

Reply:
yeah there were fixes in the plugin, it no longer emits a validation failure, hence we removed it from the develop branch sometime back.